TACC/tup-ui
# Task/tup 457 add sys queue details  content layout

## Overview

Use Dashboard layout to make new systems section accessible on small windows.

## Related

- tweaks #234
- includes #256

## Changes

See #256 until I finalize solution.

## Testing

1. Open http://localhost:8000/portal/system_status 
2. View layout on wide screen:
    1. All panels with content should stack vertically.
    2. _The panel without content should be on the fight side of those with content._
    3. All panels should have the same space between them.
    4. No extra space should exist.*
3. View layout on narrow screen:
    1. All panels (with and without content) should stack vertically.
    2. _The panel without content should be on the bottom._
    3. All panels should have the same space between them.
    4. No extra space should exist.*

<sup>* E.g. a layout-influencing gap between panel with content and panel without content.</sup>

## UI

See #256 until I finalize solution.

## Notes

1. There are problems with the #256 solution. (See it's "Notes" section.)
